app/experimenter/experiments/constants/nimbus.py

```diff
@@ -682,6 +682,7 @@ def parse(version_str):
         FIREFOX_98 = "98.!"
         FIREFOX_9830 = "98.3.0"
         FIREFOX_99 = "99.!"
+        FIREFOX_9910 = "99.1.0"
```

Comment:
Just curious to know why `99.1.0`, as in other places its `99.1`? does it makes a difference?

Reply:
It does make a difference, it's a bit tricky I can explan when next we chat.

Reply:
I added it as 99.1.0 here to follow the convention above. Looks like Jared says (below) that's a question for the person who requested the version to be added